sections/multicolumn.liquid

```diff
@@ -89,7 +97,7 @@
       </ul>
 
       {%- if section.settings.swipe_on_mobile -%}
-        <div class="slider-buttons no-js-hidden medium-hide{% if section.blocks.size < 2 %} small-hide{% endif %}">
+        <div class="slider-buttons no-js-hidden medium-hide{% if show_mobile_slider == false %} small-hide{% endif %}">
```

Comment:
If we wanted we could probably align the approach here with the other 2 I commented on or is there another consideration?

Reply:
The reason it's different here is because multicolumn only shows the slider on mobile and not tablet. So I kept `medium-hide` but updated the other statements 👍